src/main/java/com/yugabyte/sample/apps/MultiTableMultiIndexInserts.java

```diff
@@ -0,0 +1,382 @@
+    Connection conn = connectionMap.get(threadIdx);
+
+    for(int i = 0; i < rowsCount.get("a2"); ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setString(2, "\"name:de=Nowosibirsk\"");
+      preparedStatement.setInt(3, getNextInt(threadIdx));
+
+      for(int j = 4; j <= 10; ++j) {
+        Integer[] arr = new Integer[0];
+        Array sqlArr = conn.createArrayOf("int", arr);
+        preparedStatement.setArray(j, sqlArr);
+      }
+
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getDStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = preparedStatements.get(threadIdx).get(getTableName("d"));
+
+    for(int i = 0; i < rowsCount.get("d"); ++i) {
+      preparedStatement.setInt(1, getNextInt(threadIdx));
+      preparedStatement.setString(2, getRandomString());
+      preparedStatement.setInt(3, getNextInt(threadIdx));
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getA3Statement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = preparedStatements.get(threadIdx).get(getTableName("a3"));
+
+    for(int i = 0; i < rowsCount.get("a3"); ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setInt(2, getNextInt(threadIdx));
+      preparedStatement.setString(3, "");
+      preparedStatement.setString(4, "0101000020E61000002A05DD5E525953C0F74773BF9EA54640");
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getA3AStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = preparedStatements.get(threadIdx).get(getTableName("a3a"));
+
+    for(int i = 0; i < rowsCount.get("a3a"); ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setObject(2, getNextUUID(threadIdx));
+      preparedStatement.setInt(3, getNextInt(threadIdx));
+      preparedStatement.setInt(4, getNextInt(threadIdx));
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getMStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = preparedStatements.get(threadIdx).get(getTableName("m"));
+
+    for(int i = 0; i < rowsCount.get("m"); ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setString(2, "abcdefghij");
+      preparedStatement.setString(3, "abcdefghij");
+      preparedStatement.setString(4, "abcdefghij");
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getMVStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = preparedStatements.get(threadIdx).get(getTableName("mv"));
+
+    for(int i = 0; i < rowsCount.get("mv"); ++i) {
```

Comment:
- Nit: space after `for`.
- Compute the upper bound of the loop once outside of the loop.